src/dmd/expression.d

```diff
@@ -260,6 +260,14 @@ extern (C++) void expandTuples(Expressions* exps)
                     i--;
                     continue;
                 }
+                else // Expand a TypeTuple
+                {
+                    exps.remove(i);
+                    auto texps = new Expressions();
+                    foreach (a; *tt.arguments)
```

Comment:
Use setLength please and assign to the index.

Reply:
What you are doing right now is Equivalent to `~=` in a loop.
Which should be avoided if the number of elements inserted per iteration is known and the loop is bounds are known you can preallocate.

Reply:
Right, should preallocate.

Reply:
Done.